src/courtroom.cpp

```diff
@@ -1870,24 +1877,34 @@ void Courtroom::handle_chatmessage(QStringList *p_contents)
   if (f_char_id >= 0)
     f_charname = ao_app->get_showname(char_list.at(f_char_id).name);
 
-  chatlogpiece *temp =
-      new chatlogpiece(f_charname, f_displayname, m_chatmessage[MESSAGE], false,
-                       m_chatmessage[TEXT_COLOR].toInt());
-  ic_chatlog_history.append(*temp);
-  if (ao_app->get_auto_logging_enabled())
-    ao_app->append_to_file(temp->get_full(), ao_app->log_filename, true);
+  if (m_chatmessage[MESSAGE] == " ") // User-created blankpost
```

Comment:
Use [`QString::trimmed()`](https://doc.qt.io/qt-5/qstring.html#trimmed) in case user tried to use multiple spaces (e.g. old client not allowing same message to be sent twice, so they try to blankpost by adding more spaces):

```suggestion
  if (m_chatmessage[MESSAGE].trimmed().isEmpty()) // User-created blankpost
```

Reply:
What if they wanna emulate whispering with silent lip flaps tho

Reply:
Players should use ellipses for that use case; e.g.
```
...        ...         ...
```

Reply:
Good idea